cmd/shipwright-build-controller/main.go

```diff
@@ -113,3 +136,28 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+// checkForPipelinesInstalled tries to find the "TaskRun" resource on the api
+// returns (true, nil) if resource is found, (false, nil) otherwise
+// and error as second return parameter in case of failed request
+func checkForPipelinesInstalled(mgr manager.Manager) (bool, error) {
+	rm := mgr.GetRESTMapper()
+
+	_, err := rm.KindFor(pipeline.TaskRunResource.WithVersion(pipelinev1beta1.SchemeGroupVersion.Version))
+	if err != nil {
+		if meta.IsNoMatchError(err) {
+			return false, nil
+		}
+
+		return false, err
+	}
+
+	return true, nil
+}
+
+// writeTerminationMessage writes the message passed as argument
+// into the termination log file, making it a clear explanation for
+// pod failure
+func writeTerminationMessage(terminationLogPath string, msg string) error {
+	return ioutil.WriteFile(terminationLogPath, []byte(msg), 0644)
```

Comment:
I'm not sure this method adds much, compared to just calling WriteFile with a comment explaining what it does

Reply:
i fixed that!